### ISSUE TITLE
Ongoing development work in EMIF

### ISSUE THREAD

**Issue by PRijnbeek** [OHDSI/Atlas#353]
This issue is created for awareness of the enhancements for ATLAS that are currently worked on in the EMIF project. It will be updated and extended in the upcoming development cycles.

- [x] add measurement frequency output, e.g. number of people with 1,2,3,4,5,... bmi measurements

- [ ] add functionality to search on ATC code in Achilles. This means another aggregation level in R code and added option to switch between ATC and RXnorm output on the interface.

Furthermore, EMIF is working on the integration of ATLAS in the EMIF platform:

- [ ] Security layer integration

- [ ] Offline distributed querying using ATLAS, i.e. cohort creation, distribution to partners, sharing of results sets with central ATLAS instance. Each CDM owner can be the central ATLAS instance, but we are also creating a central cloud-based instance of ATLAS for this purpose.

**Comment by PRijnbeek** [comment]
Update: we have implemented a cumulative distribution of Measurement, Procedures etc. This is added in Achilles, AchillesWeb, and Atlas. Pull request will be done from our fork once our evaluation is finished.

**Comment by t-abdul-basser** [comment]
Excellent. Before that I will create a separate issue for each enhancement request/feature request, so that you can reference in commits, PRs, etc.

**Comment by anthonysena** [comment]
@PRijnbeek - apologies that this is still open after all of this time. I'm going to close this out assuming that we've either addressed this or it is no longer applicable. Please do re-open if I have this incorrect. Thanks!
